packages/video_player/video_player/lib/video_player.dart

```diff
@@ -18,8 +18,8 @@ import 'src/closed_caption_file.dart';
 export 'src/closed_caption_file.dart';
 
 final VideoPlayerPlatform _videoPlayerPlatform = VideoPlayerPlatform.instance
-  // This will clear all open videos on the platform when a full restart is
-  // performed.
+// This will clear all open videos on the platform when a full restart is
+// performed.
```

Comment:
Did `dart format` remove the indentation here, or was that changed manually?

Reply:
This was done by `dart format` I rerun dart format to double check and I do not get any changes